src/components/ui/DTable.tsx

```diff
@@ -85,10 +85,10 @@ const Row = ({ rowHeader, beginner, intermediate, advanced, expert, total, myRan
   return (
     <tr className='text-tertiary text-xs my-2'>
       <th scope='row' className='py-0.5 pr-2 text-right'>{rowHeader}</th>
-      <td className={`${highlighted ? 'dtable-highlight' : ''} ${highlighted && isInMyRange(0, myRange) ? 'dtable-my-range' : ''}`}>{beginner}</td>
-      <td className={`${highlighted ? 'dtable-highlight' : ''} ${highlighted && isInMyRange(1, myRange) ? 'dtable-my-range' : ''}`}>{intermediate}</td>
-      <td className={`${highlighted ? 'dtable-highlight' : ''} ${highlighted && isInMyRange(2, myRange) ? 'dtable-my-range' : ''}`}>{advanced}</td>
-      <td className={`${highlighted ? 'dtable-highlight' : ''} ${highlighted && isInMyRange(3, myRange) ? 'dtable-my-range' : ''}`}>{expert}</td>
```

Comment:
In this case I think it's better to update the `dtable` and `dtable-highlight` classes in  [src/styles/defaults.css](https://github.com/OpenBeta/open-tacos/blob/develop/src/styles/defaults.css#L411-L417).  

This way you preserve the original intent of the code and the DTable unit test.

Something like this

```css
table td {
  @apply my-2 text-base-content !important;
}

table td.dtable-highlight {
  @apply text-sm  bg-gray-200;
}
```

For reference you can find DaisyUI semantic color classes here:  https://daisyui.com/docs/colors/

Some of the components were developed before we adopted DaisyUI so you see TailwindUI color classes.  We should avoid that if possible.

Reply:
I did, but when I tried to save default.css the file, it gave me an error. That's why I approached it this way.